security-framework/src/secure_transport.rs

```diff
@@ -281,7 +290,15 @@ impl<S> MidHandshakeClientBuilder<S> {
                         let err = Error::from_code(errSecTrustSettingDeny);
                         return Err(ClientHandshakeError::Failure(err));
                     }
-                    TrustResult::RecoverableTrustFailure |
+                    TrustResult::RecoverableTrustFailure => {
```

Comment:
You can simplify this to `TrustResult::RecoverableTrustFailure if danger_accept_invalid_certs => { ... }` and then keep the arm below unmodified to avoid having to duplicate the not trusted stuff.

Reply:
Yes, that's much nicer looking, thanks!